san/sanbase_graphql.py

```diff
@@ -280,6 +280,30 @@ def top_holders_percent_of_total_supply(idx, slug, **kwargs):
 
     return query_str
 
+def news(idx, tag, **kwargs):
+    kwargs = _transform_query_args(**kwargs)
+
+    query_str = """
+    query_{idx}: news(
+        tag: \"{tag}\",
+        from: \"{from_date}\",
+        to: \"{to_date}\",
+        size: {size}
+    ){{
+        datetime,
+        description,
+        sourceName,
+        title,
+        url
```

Comment:
Let's rearrange the params in the order:
1. datetime
2. title
3. description
4. source
5. url

Reply:
Done